cmd/mothership/local/options.go

```diff
@@ -80,13 +72,19 @@ func componentsFromStrings(list []string, values []string) ([]*keb.Component, er
 	}
 
 	for _, item := range list {
-		s := strings.Split(item, "@")
-		name := s[0]
+		if strings.HasPrefix(item, "{") {
```

Comment:
Shouldn't this always be true?

Reply:
I added this because I wanted keep compatibility.